mindmeld/components/entity_recognizer.py

```diff
@@ -206,6 +210,18 @@ def load(self, model_path):
         self.ready = True
         self.dirty = False
 
+    def _predict(self, query, time_zone=None, timestamp=None, dynamic_resource=None):
```

Comment:
I dont think this function is necessary at all if the `_model` was constructed with the system entity recognizer.

Reply:
done